_docker/drupal/drupal-filesystem/web/modules/custom/rhd_assemblies/src/Service/DownloadManagerApi.php

```diff
@@ -33,18 +36,55 @@ class DownloadManagerApi {
    */
   protected $entityTypeManager;
 
+  /**
+   * Download Manager cache bin.
+   *
+   * @var \Drupal\Core\Cache\CacheBackendInterface
+   */
+  protected $cacheBin;
+
+  /**
+   * Logger factory service.
+   *
+   * @var \Drupal\Core\Logger\LoggerChannelFactory
+   */
+  protected $loggerFactory;
+
+  /**
+   * Config factory service.
+   *
+   * @var \Drupal\Core\Config\ConfigFactory
+   */
+  protected $configFactory;
+
   /**
    * Constructor.
    *
    * @param GuzzleHttp\ClientInterface $client
    *   The Guzzle client.
    * @param Drupal\Core\Entity\EntityTypeManager $entity_type_manager
-   *   Drupal's entity type manager.
+   *   entity type manager.
+   * @var \Drupal\Core\Cache\CacheBackendInterface $cache_bin
```

Comment:
Considering phpcs integration:

There are PHPCS warnings when this is @var, as well as the next 2.  The warnings are cleared when changing this from @var to @param. 

Line 117 exceeds the limit (80 characters) by 1.